README.md

```diff
@@ -119,7 +117,7 @@ fn code() -> Result<(), Box<Error>> {
     // we will create it. The collection creation is more complex and
     // has many options (such as indexing and so on).
     let collection = {
-        let collections = core.run(client.list_collections(&database))?;
+        let collections = core.run(client.list_collections(&DATABASE))?;
```

Comment:
`list_collections` takes the database name, not the instance. However, I wonder why this changed?